gson/src/main/java/com/google/gson/Gson.java

```diff
@@ -254,7 +254,7 @@ public Gson() {
     this(
         Excluder.DEFAULT,
         DEFAULT_FIELD_NAMING_STRATEGY,
-        Collections.<Type, InstanceCreator<?>>emptyMap(),
+        Collections.emptyMap(),
```

Comment:
Not sure if the changes here for the `Gson` constructor are good, or if without type arguments this makes the constructor call more difficult to understand.

Might depend on whether your IDE shows parameter name hints.

Reply:
I think it's fine.